### PR TITLE
 PWGGA/GammaConv/     introducing minimal output option

### PR DESCRIPTION
**Pull request by AMechler**

### CODE REVIEW

**Comment by nschmidtALICE** [comment]
+test

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx [hunk 1]
```diff
@@ -1551,27 +1563,28 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
...
-    fHistoClusOverlapHeadersGammaPt[iCut]->SetXTitle("p_{T,clus} (GeV/c), selected header w/ overlap");
-    fClusterOutputList[iCut]->Add(fHistoClusOverlapHeadersGammaPt[iCut]);
-    fHistoClusAllHeadersGammaPt[iCut]       = new TH1F("ClusGammaAllHeaders_Pt", "ClusGammaAllHeaders_Pt", nBinsClusterPt, arrClusPtBinning);
-    fHistoClusAllHeadersGammaPt[iCut]->SetXTitle("p_{T,clus} (GeV/c), all headers");
-    fClusterOutputList[iCut]->Add(fHistoClusAllHeadersGammaPt[iCut]);
-    fHistoClusRejectedHeadersGammaPt[iCut]  = new TH1F("ClusGammaRejectedHeaders_Pt", "ClusGammaRejectedHeaders_Pt", nBinsClusterPt, arrClusPtBinning);
-    fHistoClusRejectedHeadersGammaPt[iCut]->SetXTitle("p_{T,clus} (GeV/c), rejected headers");
-    fClusterOutputList[iCut]->Add(fHistoClusRejectedHeadersGammaPt[iCut]);
-
+    if(fDoLightOutput!=2){
```
Comment (FriederikeBock) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: the cluster histograms have to stay please.

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx [hunk 2]
```diff
@@ -1843,8 +1856,8 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
       fHistoTrueSecondaryConvGammaMCPt                      = new TH2F*[fnCuts];
     }
 
-    fHistoTrueClusGammaPt                           = new TH1F*[fnCuts];
-    if(!fDoLightOutput){
+    if(fDoLightOutput!=2) fHistoTrueClusGammaPt                           = new TH1F*[fnCuts];
+    if(fDoLightOutput==0){
```
Comment (FriederikeBock) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: also the true cluster histrograms should stay please

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx [hunk 3]
```diff
@@ -2360,7 +2380,7 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
         fTrueJetHistograms[iCut]->Add(fHistoUnfoldingRejectInvMassZ[iCut]);
       }
 
-      if(!fDoLightOutput){
+      if(fDoLightOutput==0){
```
Comment (FriederikeBock) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: these have to stay.

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx [hunk 4]
```diff
@@ -1536,7 +1548,7 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
     }
 
     if (  (fIsMC>0 && fDoMaterialBudgetWeightingOfGammasForTrueMesons) ) {
-      if(!fDoLightOutput){
+      if(fDoLightOutput==0){
```
Comment (FriederikeBock) [inline PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: these have to stay please too!

File: PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx (hunk 3 above)
Comment (FriederikeBock) [reply PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx]: Please also check that the dir gamma extraction will still work with this output level (ExtractSignalGammaV2.C).

**Pull request closed without merging**